plugins/rebalance/init.go

```diff
@@ -0,0 +1,62 @@
+package rebalance
+
+import (
+	"github.com/gluster/glusterd2/glusterd2/servers/rest/route"
+	"github.com/prashanthpai/sunrpc"
+)
+
+// Plugin is a structure which implements GlusterdPlugin interface
+type Plugin struct {
+}
+
+// Name returns name of plugin
+func (p *Plugin) Name() string {
+	return "rebalance"
+}
+
+// SunRPCProgram returns sunrpc program to register with Glusterd
+func (p *Plugin) SunRPCProgram() sunrpc.Program {
+	return nil
+}
+
+// RestRoutes returns list of REST API routes to register with Glusterd
+func (p *Plugin) RestRoutes() route.Routes {
+	return route.Routes{
+		route.Route{
+			Name:        "RebalanceStart",
+			Method:      "POST",
+			Pattern:     "/volumes/rebalance/{volname}/start",
+			Version:     1,
+			HandlerFunc: rebalanceStart},
+		route.Route{
+			Name:        "RebalanceStartForce",
+			Method:      "POST",
+			Pattern:     "/volumes/rebalance/{volname}/start/{force}",
```

Comment:
Would a force option need to have a different API? I guess no?

@kshlm @prashanthpai @aravindavk How are we handing the volume start force?

Reply:
I think it makes sense to have force be an argument to the start API instead a separate API.

Reply:
force should be included in the HTTP request(Not part of URL)

    {
        "force": true
    }